superset-frontend/src/dashboard/components/nativeFilters/FilterBar/FilterControls/FilterValue.tsx

```diff
@@ -195,6 +197,10 @@ const FilterValue: React.FC<FilterProps> = ({
       />
     );
   }
+  const filterState = { ...filter.dataMask?.filterState };
+  if (filterState.value === undefined && preselection) {
+    filterState.value = preselection;
```

Comment:
I think in previous PR someone added regression for validation, but because your changes affect this place also, can you add here like:
```
const isMissingRequiredValue = checkIsMissingRequiredValue(
    filter,
    filter.dataMask?.filterState,
  );
...
filterState={{
            ...filter.dataMask?.filterState,
            validateMessage: isMissingRequiredValue && t('Value is required'),
          }}
```

or may be we can create separate PR for it